index.js

```diff
@@ -1,17 +1,62 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos > this.length || pos < 0) {
+      throw new Error("OutOfBounds");
+    } else {
+      return this.items[pos];
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length > 0) {
+      return this.items[this.length - 1];
+    } else {
+      throw new Error("EmptySortedList");
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.length > 0) {
+      return this.items[0];
+    } else {
+      throw new Error("EmptySortedList");
+    }
+  }
 
-  sum() {}
+  sum() {
+    let sum = 0;
 
-  avg() {}
+    for (let i = 0; i < this.length; i++) {
+      sum += this.items[i];
+    }
```

Comment:
You could also have used an array method here! Try using `.reduce` to achieve the same goal. 💪